build/run-me.sh

```diff
@@ -57,6 +57,10 @@ dockerfile_for_project() {
         build-update-payloads)
             printf "%s\n" "common/Dockerfile"
             ;;
+        cross-compile)
```

Comment:
Can you please change it to `docker-cross-compile`. This way is more clear that it is related to the dockercross and not a general/ordinary cross-compilation task.

Reply:
You are right, I guess we can be more explicit and say this is a "dockcross" project.